internal/api/features/topics_test.go

```diff
@@ -0,0 +1,84 @@
+/*
+Copyright 2024.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package features
+
+import (
+	"encoding/json"
+	"os"
+	"reflect"
+	"testing"
+)
+
+func TestGetTopics(t *testing.T) {
+	var expected, actual TopicInfo
+
+	topicsData, err := os.ReadFile("_topics.json")
```

Comment:
bad test, because we mess with the internal implementation details (which we should not).
I'd just test the "active" topic list includes well known topics which every released operator must support. I'd actually hardcode  that list in the test code.

Reply:
done, thanks